static/app/types/organization.tsx

```diff
@@ -106,8 +108,16 @@ export type Member = {
   projects: string[];
   role: string;
   roleName: string;
-  roles: MemberRole[]; // TODO(ts): This is not present from API call
-  teams: string[];
+  roles: OrgRole[]; // Deprecated: use orgRoleList
+
+  teamRoleList: TeamRole[]; // TODO: Move to global store
+  teamRoles: {
+    role: string | null;
+    teamSlug: string;
+    isActive?: boolean;
+  }[];
+  teams: string[]; // # Deprecated, use teamRoles
```

Comment:
Can we just remove this type?

If this is a type, then we should see everywhere this is currently being used and we can sweep through and update right?

Reply:
I feel like this could be removed in a follow up PR once the migration from `teams` to `teamRoles`  has been completed. Similar to how we perform some database table migrations.